File: SL-lab/Lab8/test/prg5.py
def OddRange(num1,num2):
    odd = []
    for i in range(num1+1,num2):
        if i%2 != 0:
            odd.append(i)
    return odd

# taking the input and converting it to integer 
# input() takes the input
# int() converts it to integer

num1 = int(input())
num2 = int(input())
print(OddRange(num1,num2))

File: SL-lab/Lab8/test/prg5.js
function OddRange(num1,num2)
{
    odd = []

    for(var i=Number(num1)+1;i<num2;i++)
        if(i%2!=0)
            odd.push(i)
    return odd
}

var num1 = prompt("Enter the value for the first number")
var num2 = prompt("Enter the value for the second number")

console.log(OddRange(num1,num2))
